<launch>

	<!-- Circuit Race Controller node -->
	<node name="iarrc_circuit_race_controller" pkg="iarrc_software" type="iarrc_circuit_race_controller" output="screen" respawn="false">
		<param name="stoplight_topic" type="str" value="/light_change"/>
		<param name="drive_speed" type="int" value="8"/>
		<param name="speed_topic" type="str" value="/speed"/>
	</node>

</launch>
